pywr/notebook/__init__.py

```diff
@@ -287,10 +287,10 @@ def pywr_json_to_d3_json(model, attributes=False):
 
     nodes = []
     node_classes = create_node_class_trees()
-
     for node in model["nodes"]:
 
-        if node["type"].lower() in ["annualvirtualstorage", "virtualstorage"]:
+        if node["type"].lower() in ["annualvirtualstorage", "virtualstorage", "aggregatednode", "aggregatedstorage"]:
```

Comment:
Should we be looking up the class here and testing for `allow_isolated`?

Reply:
I had a look at this but I don't think you can test the value of `allow_isolated` or `virtual` without creating an instance of the node. In this case, the function could just load the full model and call `pywr_model_to_d3_json`